### DISCUSSION TITLE
Choose Output Name + Extension of Original File + New Extension

### DISCUSSION THREAD

**Discussion by Joelson-Forte** [Purfview/whisper-standalone-win#139] [Q&A] [answered]
Hi @Purfview 
Can you please let me know how do I add the original file extension name in the srt output file name?

For example:

Currently it looks like this:
Audio 01.wav = Audio 01.srt

I need it to stay like this
Audio 01.wav = Audio 01.wav.srt

It is possible?

**Comment by Purfview** [comment, marked as answer]
> Can you please let me know how do I add the original file extension name in the srt output file name?

You can't, there is no such option atm.
